data/json/recipes/recipe_food.json

```diff
@@ -8317,5 +8321,53 @@
       [ [ "sweet_fruit_like", 1, "LIST" ] ],
       [ [ "sugar", 5 ] ]
     ]
-  }
+  },
+  {
+    "type": "recipe",
+    "result": "sourdough_young",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREAD",
+    "skill_used": "cooking",
+    "difficulty": 6,
+    "time": 10000,
```

Comment:
Ten minutes for just mixing up water and flour in a jar?

Reply:
To do a good job of it you have to weigh it out, record weights, mark the jar, and a bunch of other minutiae. Takes me about ten minutes.

Reply:
Actually, I had splitting mature starter at 5 mins which is about right. However in checking it, I noticed that the difficulty was still way up. So fixed that.